src/models/Place.js

```diff
@@ -22,4 +23,9 @@ const placeSchema = mongoose.Schema(
   }
 )
 
+// methods which throw errors when there's nothing matching the given id
+placeSchema.statics.findByIdOrError = findByIdOrError
```

Comment:
use helper.

That way, when you add a new method, you do not need to do it everywhere..